Detectors/MUON/MCH/Base/include/MCHBase/HitTime.h

```diff
@@ -0,0 +1,41 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/** @file HitTime.h
+ * C++ Muon MCH hit time definition.
+ * @author  Andrea Ferrero, CEA
+ */
+
+#ifndef ALICEO2_MCH_BASE_HIT_TIME_H_
+#define ALICEO2_MCH_BASE_HIT_TIME_H_
+
+#include <cinttypes>
+
+namespace o2
+{
+namespace mch
+{
+
+struct HitTime {
```

Comment:
The first comment I'd make is about naming : I don't like HitTime as I feel Hit is really too much connoted "simulation". Given that struct is only used (at least in this PR) in Digit, why not make it a Time struct of the Digit itself ?

Reply:
Time definition moved inside Digit structure, as you proposed.